.github/workflows/pruner.yml

```diff
@@ -0,0 +1,12 @@
+name: Openstack Pruner
+run-name: Openstack Pruner
+on:
+  schedule:
+    - cron:  '0/5 */1 * * *'
```

Comment:
*/1 is redundant and confusing. Just do *

Reply:
also not 0/5 just 5

`'5 * * * *'`

Reply:
OK Handled